### ISSUE TITLE
`NSGenericException` in Xcode Beta 3

### ISSUE THREAD

**Issue by roblabs** [SCENEE/FloatingPanel#243]
### `NSGenericException` in Xcode Beta 3

Error while running the Maps sample app.  Not a problem with the library, but the sample app is broken

`*** Terminating app due to uncaught exception 'NSGenericException', reason: 'Access to UISearchBar's _searchField ivar is prohibited. This is an application bug'`

### Expected behavior
* No exceptions

### Actual behavior
`UISearchBar.searchTextField` is now available.  _cf._ 
* https://stackoverflow.com/questions/56667349/ios-13-custom-searchbar-with-uisearchbar-searchfield-not-working
* workaround is commenting out these lines, but won't work in production

https://github.com/SCENEE/FloatingPanel/blob/24d81a4153436f5004134f1cbaf0ab75d7d7e3b9/Examples/Maps/Maps/ViewController.swift#L152,L153

``` Swift
        let textField = searchBar.value(forKey: "_searchField") as! UITextField
        textField.font = UIFont(name: textField.font!.fontName, size: 15.0)
```

### Steps to reproduce
* Xcode 11 Beta 3 (11M362v)
* Launch Maps sample on a simulator
* Will receive the 

**Code example that reproduces the issue**

### Environment

**Library version**
```
$ git log

commit 24d81a4153436f5004134f1cbaf0ab75d7d7e3b9 (HEAD -> master, origin/master, origin/HEAD)
Merge: 5d336b9 5723a80
Author: Shin Yamamoto <shin@scenee.com>
Date:   Mon Jul 22 09:41:51 2019 +0900
```
**Installation method**

- [x] Building from Source
- [ ] CocoaPods
- [ ] Carthage
- [ ] Git submodules

**iOS version(s)**

* 13.0

**Xcode version**

* Xcode 11 Beta 3 (11M362v)

**Comment by SCENEE** [comment]
Thanks for you report! I will take care of this issue until Xcode 11 GM seed release.

**Comment by SCENEE** [comment]
I fixed this on v1.6.5. Thanks!
